frontend/src/pages/modelRegistry/screens/components/DeployRegisteredModelModal.tsx

```diff
@@ -51,16 +55,35 @@ const DeployRegisteredModelModal: React.FC<DeployRegisteredModelModalProps> = ({
     error: deployInfoError,
   } = useRegisteredModelDeployInfo(modelVersion);
 
+  const handleSubmit = React.useCallback(async () => {
+    if (!modelVersion.registeredModelId) {
+      return;
+    }
+
+    try {
+      await Promise.all([
+        modelRegistryApi.api.patchModelVersion({}, { state: ModelState.LIVE }, modelVersion.id),
+        modelRegistryApi.api.patchRegisteredModel(
+          {},
+          { state: ModelState.LIVE },
+          modelVersion.registeredModelId,
+        ),
+      ]);
+      onSubmit?.();
+    } catch (submitError) {
+      throw new Error('Failed to update timestamps after deployment');
```

Comment:
I may have missed this before - throwing an error here doesn't show it to the user, it just ends up in the browser console since there's nothing catching it that renders it (like in some places where we have errors stored in state and use them to render an Alert). That might be okay for now... we actually aren't catching errors with a lot of the patches the MR edit controls do...... 🤦 that's probably worth a tech debt issue. I'll make a note of it.